src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -21,13 +23,29 @@ export class BackupMainService implements IBackupMainService {
 	private workspacesJsonContent: IBackupWorkspacesFormat;
 
 	constructor(
-		@IEnvironmentService environmentService: IEnvironmentService
+		@IEnvironmentService environmentService: IEnvironmentService,
+		@ILifecycleService lifecycleService: ILifecycleService
 	) {
 		this.backupHome = environmentService.backupHome;
 		this.workspacesJsonPath = environmentService.backupWorkspacesPath;
+
+		if (lifecycleService) {
```

Comment:
Pushed a change to do this, I needed to create an `IVSCodeWindow` interface as well as `VSCodeWindow` imports from `'electron'`.